actions/gallery_actions.js

```diff
@@ -93,7 +95,6 @@ export const getPhotosFromCameraroll = () => async (dispatch, getState) => {
                     lon: item.node.location.longitude,
                     date: item.node.timestamp,
                     selected: false,
-                    picked_up: false,
```

Comment:
Because this is when we load geotagged cameraroll/gallery images to state, we dont need to add picked_up here..
We add `pickedUp` in `ADD_IMAGE` i.e when we select gallery image